governance/default.go

```diff
@@ -423,6 +423,7 @@ func NewGovernanceInitialize(chainConfig *params.ChainConfig, dbm database.DBMan
 	ret := NewGovernance(chainConfig, dbm)
 	// nil is for testing or simple function usage
 	if dbm != nil {
+		ret.ReadGovernanceState()
```

Comment:
Can I ask why you move this line to here?

Reply:
The `ReadGovernanceState()` first sets the current set, and the `initializeCache()` overwrites the current set if necessary.